statick_tool/rsc/profile.yaml

```diff
@@ -1,4 +1,4 @@
-default: sei_cert
+default: default
```

Comment:
It might be done already, but I would like it if we maintain an easy way to run the `sei_cert` level. Maybe by adding a `profile-sei-cert.yaml` file that has the default level as `sei_cert`? I want an easy upgrade path for users that were previously using the default level of `sei_cert`. This case should be supported by a unit test.

Reply:
That profile does exist already: https://github.com/sscpac/statick/blob/main/statick_tool/rsc/sei_cert.yaml
I'll add a unit test, should I also detail this in the changelog?

Reply:
I think it would be nice to add an upgrade path to the changelog. I can help if you want me to.

Reply:
[c4a97ab](https://github.com/sscpac/statick/pull/435/commits/c4a97ab1b2fff9d2f60582964598cfd96cb9e1ba)